Chap_API_Struct.tex

```diff
@@ -4776,12 +4776,20 @@ \subsection{Query attributes}
 
 %
 \declareAttributeNEW{PMIX_QUERY_NUM_PSETS}{"pmix.qry.psetnum"}{size_t}{
-Return the number of psets defined in the specified range (defaults to session).
+Return the number of process sets defined in the specified range (defaults
+to session).
 }
 
 %
 \declareAttributeNEW{PMIX_QUERY_PSET_NAMES}{"pmix.qry.psets"}{char*}{
-Return a comma-delimited list of the names of the psets defined in the specified range (defaults to session).
+Return a comma-delimited list of the names of the process sets defined in the
+specified range (defaults to session).
+}
```

Comment:
During the working group discussion we made this note (per https://github.com/pmix/pmix-standard/issues/254):
```
When querying the process set names for a given process it is possible to get either:
 * A single string if it is in exactly one process set
 * An array of strings if it is in more than one process set
```

I vaguely remember that we wanted an array because we wanted to avoid the limitation of reserving a special character in the string. However, returning a comma-delimited list only reservers the `,` character which might be fine. I just wanted to bring it up in case others remember better the context behind that note.

Reply:
It gets burdensome to coders if we make the return datatype ambiguous, so let's just make it always be a pmix_data_array_t of char*, per latest commit

Reply:
That sounds good to me.